### DISCUSSION TITLE
如果只在一个平台，一种显卡中使用深度学习，那么tvm的优势是什么？

### DISCUSSION THREAD

**Discussion by gaoxt1983** [mlc-ai/mlc-zh#115] [General]
例如：我只在centos7基于A30，使用torchServe部署，那么tvm相对torchserve的优势是什么？

**Comment by Hzfengsy** [comment]
不是特别清楚torchserve和Torch的区别。根据我们的实验，TVM对比Torch的会有1.3x-4.5x的性能提升。（结果可能会根据模型和显卡型号不同有影响）
